src/background/services/background.ts

```diff
@@ -286,6 +290,15 @@ export class Background {
 
             // endregion
 
+            // region App
+            case 'GET_APP_CONTEXT_DATA':
+              return success(
+                await this.monetizationService.getPopupData(
+                  await this.windowState.getCurrentTab(),
+                ),
```

Comment:
We don't need all this data. Can add a new function (ok within background.ts ok) to `getAppData()`